### PR TITLE
Add script for extracting v2 translation to csv

### PR DESCRIPTION
**Pull request by tung2744**
ref #3656

Expected usage:
export: `npm run -s export-v2-translations > translation.csv`
import: `cat translation.csv | npm run import-v2-translations`

### CODE REVIEW

File: scripts/npm/export-v2-translations.mjs [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  values: { "output-file": outputFile },
+} = parseArgs({
+  options: {
+    "output-file": {
+      short: "f",
+      type: "string",
+      default: "output/v2-translations.csv",
+    },
+  },
+});
```
Comment (louischan-oursky) [inline scripts/npm/export-v2-translations.mjs:21]: Remove the -f option and output to stdout.

Comment (tung2744) [reply scripts/npm/export-v2-translations.mjs:21]: @louischan-oursky 

If running it by npm, there will be some logs in stdout, which pollute the output file.
```
> node@0.1.0 export-v2-translations
> node export-v2-translations.mjs
```

Comment (louischan-oursky) [reply scripts/npm/export-v2-translations.mjs:21]: Try `npm run -s <script>`

Comment (tung2744) [reply scripts/npm/export-v2-translations.mjs:21]: Then the script must run with specific argument to work, which seems a bit weird?

Comment (louischan-oursky) [reply scripts/npm/export-v2-translations.mjs:21]: I am very surprised that npm run its output to stdout, not stderr.

**Pull request merged**